src/docs/customize/theming.mdx

```diff
@@ -6,6 +6,10 @@ route: /customize/theming
 
 # Theming
```

Comment:
All select boxes are not openable. It is little bit confusing.

Reply:
OK, I was just testing if anybody noticed that…